sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -1,48 +1,81 @@
-import { InstitutionPrimaryContact } from "../../../types";
+import { Allow } from "class-validator";
 import { InstitutionLocationInfo } from "../../../database/entities/institution-location.model";
+import { AddressInfoOutDTO } from "../../models/common.dto";
 
 export class GetInstitutionLocationDto {
   id: number;
   data: InstitutionLocationInfo;
   name: string;
 }
 /**
- * Interface which defines the contract of how institution location data is passed to form.io
+ * Payload/Input DTO for institution location.
+ ** Class validators are not used for DTO validation
+ ** because of dry-run validation.
  */
-export interface InstitutionLocationTypeDto {
+export class InstitutionLocationFormAPIInDTO {
+  @Allow()
```

Comment:
some of these fields are mandatory fields in the form right? so, we can use ` @IsNotEmpty()` right? let me know if I am missing anything

Reply:
As mentioned in comment, as we do dryrun no class validators were added to this DTO. 
![image](https://user-images.githubusercontent.com/54600590/163024947-c1c5ab5b-635d-4ded-8da2-782ef7375bb4.png)